tests/unit_tests/test_geometry.py

```diff
@@ -362,8 +362,13 @@ def get_cyl_cell(r1, r2, z1, z2, fill):
     clad = get_cyl_cell(r1, r2, z1, z2, m2)
     water = get_cyl_cell(r2, r3, z1, z2, m3)
     root = openmc.Universe(cells=[fuel, clad, water])
-    geom = openmc.Geometry(root)
-    geom.merge_surfaces=True
+    geom = openmc.Geometry(root=root, merge_surfaces=True, surface_precision=11)
+    assert geom.merge_surfaces is True
+    geom.merge_surfaces = False
+    assert geom.merge_surfaces is False
+    assert geom.surface_precision == 11
+    geom.surface_precision = 10
+    assert geom.surface_precision == 10
```

Comment:
perhaps these additional tests are a bit unnecessary now we don't have the **kwarg? Happy to remove if that is for the best